tutorials/clustering.md

```diff
@@ -0,0 +1,76 @@
+# Tutorial: Scaling out TimescaleDB
+
+>:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
+This method of deployment is not meant for production use.
+
+TimescaleDB can be run in a multi-node setup, with one primary access node distributing
+writes and queries to multiple data nodes. For more information on the actual
+underlying architecture, you can view our [architecture][architecture] section.
+
+In this tutorial, we will show how you can create a distributed hypertable within
+a distributed database. For simplicity, we will set up one primary access node
+and two data nodes all on local host. However, in the case of scaling out, you would
```

Comment:
This example doesn't actually create the data nodes on the local host.  If wanted to do this you should use `host => 'localhost'` in your add_data_node below.  Also, the prerequisites also indicate you want to create a real cluster of three machines here, which I think is probably the correct approach.

Reply:
Furthermore, if localhost is used, then I believe new database names have to be provided.

Reply:
Lol, this was a total legacy mistake. Gonna update.

Reply:
I think it's simpler in this case to just tell people to spin up three instances of TimescaleDB